---
layout: content
title: Data
display-title: Data
permalink: /data/
---

## Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis

In this work, we performed a single-cell RNAseq timecourse of early zebrafish embryogenesis (38,731 cells, across 12 timepoints spanning 3.3 -- 12 hours post-fertilization), and reconstructed a branching specification tree leading to 25 cell types.

### Browsing the data

1. **Browse the data interactively online** in the [Broad Single-cell Portal](https://portals.broadinstitute.org/single_cell/study/single-cell-reconstruction-of-developmental-trajectories-during-zebrafish-embryogenesis)). This allows plotting the expression of genes of interest on the 3D force-directed layout, tSNE projection, or spatially assigned 50% epiboly cells.

2. **Download and [install URD](https://github.com/farrellja/URD/blob/master/INSTALL.md)**, then [download the zebrafish data as a pre-processed URD object](https://portals.broadinstitute.org/single_cell/data/public/single-cell-reconstruction-of-developmental-trajectories-during-zebrafish-embryogenesis?filename=URD_Zebrafish_Object.rds) (requires logging into the Broad Single-cell Portal). This allows more responsive plotting on the 3D force-directed layout, the dendrogram layout, or the tSNE projections (including dual-color plotting of multiple genes), as well as isolation of particular cell populations, differential expression, and more.

### Getting the data

1. **The processed [genes x cells counts matrix](https://portals.broadinstitute.org/single_cell/data/private/single-cell-reconstruction-of-developmental-trajectories-during-zebrafish-embryogenesis?filename=URD_Dropseq_Expression_Log2TPM.txt.gz)** (requires logging into the Broad Single-cell Portal) can be used as a starting point for your own analysis.

2. Alternatively, **BAM files** are deposited in NCBI GEO under accession number GSE106587. 