---
layout: content_news
title: News (Pre-2020)
display-title: Pre-2020
permalink: /news-older/
---

## New preprint: Emergence of neuronal diversity during vertebrate brain development
### November 12, 2019

Bushra profiled over 200,000 cells from zebrafish brains during 12 developmental timepoints, spanning 12 hours to 15 days post-fertilization. Through clustering and annotation, she generated an incredibly comprehensive atlas of zebrafish brain development. Bushra and Jeff worked together to profile transcriptional trajectories and the gene expression cascades during the development of the retina and hypothalamus.  Examination of the major gene expression programs in progenitor cells revealed that embryonic neural progenitors are primarily defined by spatial gene expression signatures, but that signature becomes dampened in long-term progenitors. The trajectories highlighted an interesting difference between fish and mammalian retinal development: namely that fish Muller glia cells seem to become transcriptionally distinct much earlier in development than mammalian ones. Additionally, our pseudotime analysis revealed different progenitor strategies between these two tissues. In the retina, late-stage progenitor cells transcriptionally resemble those from earlier embryonic development, while hypothalamic progenitors change over developmental time, and later-stage hypothalamic progenitors do not transcriptionally resemble those of the embryo.

Check it out in [biorxiv](https://www.biorxiv.org/content/10.1101/839860v1.full)!

----------------------------------

## New publication: Stem cell differentiation trajectories in <i>Hydra</i> resolved at single-cell resolution
### July 26, 2019

"Stem cell differentiation trajectories in <i>Hydra</i> resolved at single-cell resolution" was published today in Science! This work was a collaboration between Jeff and the lab of Celina Juliano, including Stefan Siebert, Jack Cazet, Yash Abeykoon, and Abby Primack. 

Adult <i>Hydra</i> continually renew all cells from three distinct stem cell populations. We sequenced 25,000 cells from adult <i>Hydra</i> and used URD to construct the differentiation trajectories of all cell types and identified the transcription factors expressed along each trajectory.  Surprisingly, we found that neurons and gland cells transit through a common progenitor state, and characterized the full complement of neurons, providing the first genetic handles for the endodermal nerve net. 

This work demonstrated the applicability of URD in multiple contexts, including in a case where there was no prior knowledge of cells' temporal sequence. URD version 1.1 accompanied this paper, and included new functions for using NMF module analysis to detect and remove doublets, new functions for looking at gene expression relative to pseudotime, and new plotting functions for inspecting data.

Read it at <a href="https://science.sciencemag.org/content/365/6451/eaav9314">Science</a>! Also, check out the perspective articles describing the work, including <a href="https://www.science.org/doi/10.1126/science.aay3660">"The cells of regeneration"</a> and "Looking at <i>Hydra</i> cells one at a time".

----------------------------------

## Jeff wins Chi-Bin Chien Award from International Zebrafish Society
### June 13, 2019

The International Zebrafish Society (IZFS) selected Jeff as the 2019 winner of the [Chi-Bin Chien Award](https://www.izfs.org/page/CBCAwardNew), in memory of Dr. Chi-Bin Chien. The award rewards an outstanding graduate student, postdoc or recent faculty appointee from any country for "significant contributions to the field of zebrafish research, and exhibition of generosity and openness by a future leader in the zebrafish field."

----------------------------------

## Jeff accepts position at NICHD
### May 8, 2019

Jeff accepted a position to open his lab in early 2020 in the NICHD Intramural Research program, with a joint appointment in NHLBI.

----------------------------------

## <i>Science</i> 2018 Breakthrough of the Year
### December 20, 2018

Our image was the cover for tracking development cell-by-cell!

The [2018 Breakthrough of the Year](https://vis.sciencemag.org/breakthrough2018/) was chosen by *Science* magazine as "Development cell by cell." This collection of 11+ studies focused on emerging approaches to use single-cell genomics to reconstruct developmental trajectories and lineage during the development of several organisms, including zebrafish, frogs, mice, planarians, axolotl, and nematodes. 

Our work using single-cell genomics to profile early development in zebrafish, develop the software URD, and reconstruct the trajectories of early embryogenesis was one of the featured studies. We are very excited that our image was chosen to represent this amazing field on the cover. 

Read more at [Science](https://vis.sciencemag.org/breakthrough2018/), or read the paper: [Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis](https://www.ncbi.nlm.nih.gov/pubmed/29700225)

----------------------------------

## New publication: Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis
### April 26, 2018

All animals begin as a single cell that must proliferate and eventually differentiate into all distinct types of cells that make a functional animal. The genetic control of cell type specification during development has been investigated for decades, but classic studies in the field have had to focus on particular cell fate decisions and a handful of genes at a time. Recent technological advances enabled high-throughput single-cell RNA sequencing, where the complement of genes activated in individual cells can be profiled. We used this approach to generate a 38,000-cell single-cell RNAseq timecourse of early zebrafish embryogenesis across 12 stages spanning 9 hours of development. This period of time begins just after embryonic cells begin to show differences in their gene expression and ends when dozens of distinct cell types can be recognized by their morphologies or expression of distinct marker genes.

We then developed a computational technique, URD, that looked for cells that had very similar gene expression and used those connections to uncover the paths through gene expression that cells take as they adopt their specific fates. This approach generated a branching tree that described the molecular specification of 25 different cell types in the zebrafish embryo. Study of the branching tree revealed how cells change their gene expression as they become specialized, recovering both classic and new markers of cell populations, and suggesting candidate regulators of cell specification events. Additionally, by profiling cells from a mutant that lacks a developmental signal at an early timepoint, we found that mutant cells adopted a subset of wild-type gene expression states, while no new cell states were found. However, we could predict the tissues that would later be missing in the mutant based on the loss of early cell states. Finally, we found that at one developmental branchpoint, after cells have become two transcriptionally distinct progenitor pools, there were cells that unexpectedly seemed to switch their specification from one cell type to another.

Overall, these approaches and findings provide a rich resource for use by others in the community and lay the foundation for many of the directions of the lab.

Read it here: [Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis](https://www.ncbi.nlm.nih.gov/pubmed/29700225)

Also, check out a perspective article about our research: [&quot;A new view of embryo development and regeneration&quot;](https://science.sciencemag.org/content/360/6392/967.long)
