---
layout: content
title: Publications
display-title: Key Publications
permalink: /publications/
---

A full list of publications is available at [Pubmed](https://www.ncbi.nlm.nih.gov/myncbi/jeffrey%20a..farrell.1/bibliography/public/).

<div class="pub-block">
<b>Gene module reconstruction elucidates cellular differentiation processes and the regulatory logic of specialized secretion</b><br />
Wang Y, Liu J, Du LY, Wyss JL, <u>Farrell JA and Schier AF (equal contribution)</u> .<br />

<i>bioRxiv</i>. 2023 Dec 29; doi:10.1101/2023.12.29.573643<br />

[<a href="https://www.biorxiv.org/content/10.1101/2023.12.29.573643">bioRxiv</a>]

<p class="pub-desc">
We integrated gene expression dynamics from single-cell RNAseq developmental trajectories with functional annotations to generate gene modules that describe differentiation. When applied to the zebrafish axial mesoderm, these revealed known and newly associated differentiation processes (as well as their temporal ordering), including the anticipatory deployment of the unfolded protein response (UPR) in notochord and hatching gland differentiation. By profiling loss- and gain-of-function embryos, we found that the UPR transcription factors <i>creb3l1</i>, <i>creb3l2</i>, and <i>xbp1</i> are master regulators of a general secretion program. <i>creb3l1/creb3l2</i> additionally activate an extracellular matrix secretion program, while Xbp1 partners with Bhlha15 to activate a gland-specific secretion program. This reveals how the regulatory logic by which these related cell types activate shared and cell-type-specific secretory programs.
</p>

<img src="../assets/news/2023-12-29-Wang.png" alt="Wang et al. Fig 3" class = "pubimg" width = "350px" />
</div>



<div class="pub-block">
<b>Single-cell analysis of shared signatures and transcriptional diversity during zebrafish development</b><br />
Sur A, Wang Y, Capar P, Margolin G, Prochaska M, <u>Farrell JA</u>.<br />

<i>Developmental Cell</i>. 2023 Dec 18; doi:10.1016/j.devcel.2023.11.001<br />
<i>bioRxiv</i>. 2023 Mar 21; 58 doi:10.1101/2023.03.20.533535<br />

[<a href="https://pubmed.ncbi.nlm.nih.gov/37995681/">Pubmed</a>] 
[<a href="https://www.sciencedirect.com/science/article/pii/S1534580723005774">Developmental Cell</a>]
[<a href="https://www.biorxiv.org/content/10.1101/2023.03.20.533545v3">bioRxiv</a>]

<p class="pub-desc">
We generated a 489,000 cell atlas of 62 stages of zebrafish development that extends until the fish are freely swimming and hunting. Globally, we identified the long-term transcriptional states during development and the gene expression programs that are reused across cell types. Additionally, focused analyses within particular tissues revealed several surprising cell states, including distinct cell populations within intestinal smooth muscle, distinct pericyte subtypes, and zebrafish homologs of <i>best4+</i> intestinal cells. We confirmed all populations using <i>in situ</i> hybridization, built transcriptional trajectories to describe their likely developmental process, and predicted candidate regulators of each cell type. To make this work accessible, we generated the online resource, <a href="https://daniocell.nichd.nih.gov/">Daniocell</a> for browsing zebrafish single-cell RNAseq data.</p>

<img src="../assets/news/2023-03-21-Sur.png" alt="Sur et al. Fig 1" class = "pubimg" width = "350px" />
</div>



<div class="pub-block">
<b>Single-cell temporal dynamics reveals the relative contributions of transcription and degradation to cell-type specific gene expression in zebrafish embryos</b><br />
Fishman L, Nechooshtan G, Erhard F, Regev A, <u>Farrell JA and Rabani M (equal contribution)</u>.<br />

<i>bioRxiv</i>. 2023 Apr 21; doi:10.1101/2023.04.20.537620<br />

[<a href="https://pubmed.ncbi.nlm.nih.gov/37131717/">Pubmed</a>]
[<a href="https://www.biorxiv.org/content/10.1101/2023.04.20.537620v1">bioRxiv</a>]

<p class="pub-desc">
Using combined metabolic labeling and single-cell RNAseq we were able to distinguish between maternally loaded and zygotically transcribed mRNA in zebrafish embryos. This allowed us to quantify the rates of destruction of maternally loaded mRNAs during the maternal-to-zygotic transitions and identify that differential degradation of maternal mRNAs contributes to cell type-specific gene expression programs in cells of the enveloping layer as well as the primordial germ cells. We also identified putative sequence elements in their UTRs that may mediate their different rates of destruction.
</p>

<img src="../assets/news/2023-04-21-Fishman.png" alt="Fishman et al Graphical Abstract" class="pubimg" width = "200px"/>
</div>



<div class = "pub-block">
<b>Emergence of neuronal diversity during vertebrate brain development.</b><br />
Raj B, <u>Farrell JA</u>, McKenna A, Leslie JL, Schier AF.<br />

<i>Neuron</i>. 2020; Oct 6;S0896-6273(20)30747-9; doi: 10.1016/j.neuron.2020.09.023.<br />
<i>biorxiv</i>. 2019; 839860; doi: 10.1101/839860<br />

[<a href="https://pubmed.ncbi.nlm.nih.gov/33068532/">Pubmed</a>]
[<a href="https://www.sciencedirect.com/science/article/pii/S0896627320307479">Neuron</a>]
[<a href="https://www.biorxiv.org/content/10.1101/839860v1">bioRxiv</a>]

<p class="pub-desc">

We sequenced and analyzed 200,000 cells from 12 stages of brain development (12 hours to 5 days), generating an atlas of zebrafish brain development. Analysis of the transcriptional trajectories and the gene expression cascades during the development of the retina and the hypothalamus revealed different progenitor strategies between these two tissues, and revealed that fish Muller glia cells seem to become transcriptionally distinct much earlier in development than mammalian ones.</p>

<img src="../assets/news/2020-10-16-neuron.png" alt="Raj et al Graphical Abstract" class="pubimg" width = "300px" />
</div>

<div class = "pub-block">
<b>Stem cell differentiation trajectories in Hydra resolved at single-cell resolution.</b><br />
Siebert S, <u>Farrell JA</u>, Cazet J, Abeykoon Y, Primack A, Schnitzler C, Juliano CE.<br />

<i>Science</i>. 2019; 365(6451), eaav9314; doi: 10.1126/science.aav9314<br />
<i>biorxiv</i>. 2018; 460154; doi: 10.1101/460154<br />

[<a href="https://pubmed.ncbi.nlm.nih.gov/31346039/">Pubmed</a>]
[<a href="https://science.sciencemag.org/content/365/6451/eaav9314">Science</a>]
[<a href="https://www.biorxiv.org/content/10.1101/460154v1">bioRxiv</a>]

<p class="pub-desc">
Adult Hydra continually renew all cells from three distinct stem cell populations. We sequenced 25,000 cells from adult Hydra and used URD to construct the differentiation trajectories of all cell types and identified the transcription factors expressed along each trajectory.  Surprisingly, we found that neurons and gland cells transit through a common progenitor state, and characterized the full complement of neurons, providing the first genetic handles for the endodermal nerve net. This work demonstrates the applicability of URD in multiple contexts.</p>

<p class="pub-desc">
Featured in <a href="https://www.science.org/doi/10.1126/science.aay3660">Science</a>.
</p>

<img src="../assets/news/2019-Siebert.png" alt="Siebert et al. Fig 1" class = "pubimg" width = "300px" />
</div>

<div class = "pub-block">
<b>Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis.</b><br />
<u>Farrell JA and Wang Y (equal contribution)</u>, Riesenfeld SJ, Shekhar K, Regev A<sup>†</sup>, Schier AF<sup>†</sup>.<br />

<i>Science</i>. 2018; 360 eaar3131.<br />

[<a href="https://www.ncbi.nlm.nih.gov/pubmed/29700225/">Pubmed</a>]
[<a href="https://www.science.org/doi/10.1126/science.aar3131">Science</a>]

<p class="pub-desc">We performed a 38,000-cell single-cell RNAseq timecourse of early zebrafish embryogenesis and developed a computational approach (URD) to reconstruct the gene expression trajectories in the form of a branching tree. We identified the molecular cascade leading to 25 cell types, the spatial origins of those cell types in the blastoderm, profiled a developmental signaling mutant, and identified cells that change their specification during gastrulation. We released the full data set and an accompanying open-source software package (URD).</p>

<p class="pub-desc">
Highlighted in <a href="https://www.science.org/doi/10.1126/science.aat8413">Science</a>, <a href="https://www.nature.com/articles/d41586-018-04986-5">Nature</a>, as part of <a href="https://vis.sciencemag.org/breakthrough2018/">Science's 2018 Breakthrough of the Year</a>, and in the 13th edition of the textbook <a href="https://global.oup.com/ushe/product/developmental-biology-9780197574591">Developmental Biology</a>.
</p>

<img src="../assets/news/2018-Science-Cover-1.jpg" alt="Science June 1 2018 Cover" class = "pubimg" width = "200px" />
<img src="../assets/news/2018-Science-Cover-Breakthrough.jpg" alt="Science December 21 2018 Cover Breakthrough of the Year" class = "pubimg" width = "200px" />
<img src="../assets/news/2018-Science-Fig1.png" alt="Farrell et al 2018 Figure 1" class = "pubimg" width = "300px" />
</div>

<div class = "pub-block">
<b>Spatial reconstruction of single-cell gene expression data.</b><br />
<u>Satija R and Farrell JA (equal contribution)</u>, Gennert D, Schier AF<sup>†</sup> and Regev A<sup>†</sup>.<br />

<i>Nature Biotechnology</i>. 2015; 33(5):495–502.<br />

[<a href="https://www.ncbi.nlm.nih.gov/pubmed/25867923/">Pubmed</a>]
[<a href="https://www.nature.com/articles/nbt.3192">Nature Biotechnology</a>]

<p class="pub-desc">
We developed a novel technique to computationally assign single-cell transcriptomes from dissociated tissue to their original spatial location. We pioneered our technique on zebrafish embryos just prior to gastrulation, and thereby created a digital genome-wide expression map, identified a putative new cell state, and further characterized known progenitors. The corresponding open-source software package introduced here (<a href="https://satijalab.org/seurat/">Seurat</a>) is widely used.
</p>

<p class="pub-desc">
Highlighted in <a href="https://www.nature.com/articles/nbt.3219">Nature Biotechnology</a> and as part of <a href="https://www.nature.com/collections/gehbfbgdij">Nature Biotechnology's 25th Anniversary 25 Landmark Papers</a>.
</p>

<img src="../assets/news/2015-seurat-fig1.png" alt="Satija et al 2015 Fig 1" class = "pubimg" width = "350px" />
</div>


<div class = "pub-block">
<b>From egg to gastrula: how the cell cycle is remodeled during the Drosophila mid-blastula transition.</b><br />
<u>Farrell JA</u> and O'Farrell PH.<br />

<i>Annual Reviews in Genetics</i>. 2014; 48:269–94.<br />

[<a href="https://pubmed.ncbi.nlm.nih.gov/25195504/">Pubmed</a>]
[<a href="https://www.annualreviews.org/doi/10.1146/annurev-genet-111212-133531">Annual Reviews</a>]

<p class="pub-desc">
A review of the core mechanisms and triggers that alter cell cycle regulation at the Drosophila mid-blastula transition.
</p>

<img src="../assets/news/2014-AnnualReview-Fig1.png" alt="Farrell and O'Farrell 2014 Figure 1" class = "pubimg" width = "400px" />
</div>


<div class = "pub-block">
<b>Mechanism and Regulation of Cdc25/Twine Protein Destruction in Embryonic Cell Cycle Remodeling.</b><br />
<u>Farrell JA</u> and O'Farrell PH.<br />

<i>Current Biology</i>. 2013; 23(2):118-126.<br/>

[<a href="https://www.ncbi.nlm.nih.gov/pubmed/23290551/">Pubmed</a>]
[<a href="https://www.sciencedirect.com/science/article/pii/S0960982212013802">Current Biology</a>]

<p class="pub-desc">
Downregulation of Cdc25 activity at the Drosophila mid-blastula transition is critical in order to remodel cell cycle progression. This study identified that, contrary to previous descriptions, this is accomplished via the regulated proteolysis of Twine, a Cdc25 homolog, rather than the removal of cdc25 mRNA. The proteolysis of Twine is triggered by the onset of zygotic transcription.
</p>

<img src="../assets/news/2013-Farrell-GraphicalAbstract.jpg" alt="Farrell and O'Farrell 2013 Graphical Abstract" class = "pubimg" width = "250px" />
</div>

<div class = "pub-block">
<b>Embryonic onset of late replication requires Cdc25 down-regulation.</b><br />
<u>Farrell JA</u>, Shermoen AW,  Yuan K, O'Farrell PH.<br />

<i>Genes and Development</i>. 2012; 26(7): 714-725.<br />

[<a href="https://www.ncbi.nlm.nih.gov/pubmed/22431511/">Pubmed</a>]
[<a href="https://genesdev.cshlp.org/content/26/7/714.long">Genes and Development</a>]

<p class="pub-desc">
This study established a role for Cdk1 in regulating DNA replication during development. We showed that Cdk1 downregulation at the mid-blastula transition (through the downregulation of Cdc25) is responsible for dramatically lengthening S-phase. This identified a surprising developmental role for Cdk1 (as replication is normally driven by Cdk2) and provides insight into the outstanding question of why different regions of the genome replicate at different times.
</p>

<img src="../assets/news/2012-Farrell-Cover.gif" alt="Genes & Development April 2012 Cover" class = "pubimg" width = "300px" />
</div>


