---
layout: content
title: Publications
display-title: Key Publications
permalink: /publications/
---

A full list of publications is available at [Pubmed](https://www.ncbi.nlm.nih.gov/myncbi/jeffrey%20a..farrell.1/bibliography/public/).

**[Stem cell differentiation trajectories in Hydra resolved at single-cell resolution.](https://science.sciencemag.org/content/365/6451/eaav9314)**<br />
**Siebert S, <u>Farrell JA</u>, Cazet J, Abeykoon Y, Primack A, Schnitzler C, Juliano CE.**<br />
**<i>Science</i>. 2019; 365(6451), eaav9314; doi: 10.1126/science.aav9314**<br />
<span class="pub-desc">Adult Hydra continually renew all cells from three distinct stem cell populations. We sequenced 25,000 cells from adult Hydra and used URD to construct the differentiation trajectories of all cell types and identified the transcription factors expressed along each trajectory.  Surprisingly, we found that neurons and gland cells transit through a common progenitor state, and characterized the full complement of neurons, providing the first genetic handles for the endodermal nerve net. This work demonstrates the applicability of URD in multiple contexts.</span>

**[Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis.](https://www.ncbi.nlm.nih.gov/pubmed/29700225/)**<br />
**<u>Farrell JA and Wang Y (equal contribution)</u>, Riesenfeld SJ, Shekhar K, Regev A†, Schier AF†.**<br />
**<i>Science</i>. 2018; 360 eaar3131.**<br />
We performed a single-cell RNAseq timecourse of early zebrafish embryogenesis and developed a computational approach (URD) to reconstruct the gene expression trajectories in the form of a branching tree. We identified the molecular cascade leading to 25 cell types, the spatial origins of those cell types in the blastoderm, profiled a developmental signaling mutant, and identified cells that change their specification during gastrulation. We released the full data set and an accompanying open-source software package (URD).

**[Spatial reconstruction of single-cell gene expression data.](https://www.ncbi.nlm.nih.gov/pubmed/25867923/)**<br />
**<u>Satija R and Farrell JA (equal contribution)</u>, Gennert D, Schier AF† and Regev A†.**<br />
**<i>Nature Biotechnology</i>. 2015; 33(5):495–502.**<br />
We developed a novel technique to computationally assign single-cell transcriptomes from dissociated tissue to their original spatial location. We pioneered our technique on zebrafish embryos just prior to gastrulation, and thereby created a digital genome-wide expression map, identified a putative new cell state, and further characterized known progenitors. The corresponding open-source software package (Seurat) is widely used.

**[Mechanism and Regulation of Cdc25/Twine Protein Destruction in Embryonic Cell Cycle Remodeling.](https://www.ncbi.nlm.nih.gov/pubmed/23290551/)**<br />
**<u>Farrell JA</u> and O'Farrell PH.**<br />
**<i>Current Biology</i>. 2013; 23(2):118-126.**<br/>
Downregulation of Cdc25 activity at the Drosophila mid-blastula transition is critical in order to remodel cell cycle progression. This study identified that, contrary to previous descriptions, this is accomplished via the regulated proteolysis of Twine, a Cdc25 homolog, rather than the removal of cdc25 mRNA. The proteolysis of Twine is triggered by the onset of zygotic transcription.

**[Embryonic onset of late replication requires Cdc25 down-regulation.](https://www.ncbi.nlm.nih.gov/pubmed/22431511/)**<br />
**<u>Farrell JA</u>, Shermoen AW,  Yuan K, O'Farrell PH.**<br />
**<i>Genes and Development</i>. 2012; 26(7): 714-725.**<br />
This study established a role for Cdk1 in regulating DNA replication during development. We showed that Cdk1 downregulation at the mid-blastula transition (through the downregulation of Cdc25) is responsible for dramatically lengthening S-phase. This identified a surprising developmental role for Cdk1 (as replication is normally driven by Cdk2) and provides insight into the outstanding question of why different regions of the genome replicate at different times.