---
layout: post
title: "&quot;Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis&quot; published in <i>Science</i>"
description: ""
date: 2018-04-26
---

All animals begin as a single cell that must proliferate and eventually differentiate into all distinct types of cells that make a functional animal. The genetic control of cell type specification during development has been investigated for decades, but classic studies in the field have had to focus on particular cell fate decisions and a handful of genes at a time. Recent technological advances enabled high-throughput single-cell RNA sequencing, where the complement of genes activated in individual cells can be profiled. We used this approach to generate a 38,000-cell single-cell RNAseq timecourse of early zebrafish embryogenesis across 12 stages spanning 9 hours of development. This period of time begins just after embryonic cells begin to show differences in their gene expression and ends when dozens of distinct cell types can be recognized by their morphologies or expression of distinct marker genes.

We then developed a computational technique, URD, that looked for cells that had very similar gene expression and used those connections to uncover the paths through gene expression that cells take as they adopt their specific fates. This approach generated a branching tree that described the molecular specification of 25 different cell types in the zebrafish embryo. Study of the branching tree revealed how cells change their gene expression as they become specialized, recovering both classic and new markers of cell populations, and suggesting candidate regulators of cell specification events. Additionally, by profiling cells from a mutant that lacks a developmental signal at an early timepoint, we found that mutant cells adopted a subset of wild-type gene expression states, while no new cell states were found. However, we could predict the tissues that would later be missing in the mutant based on the loss of early cell states. Finally, we found that at one developmental branchpoint, after cells have become two transcriptionally distinct progenitor pools, there were cells that unexpectedly seemed to switch their specification from one cell type to another.

Overall, these approaches and findings provide a rich resource for use by others in the community and lay the foundation for many of the directions of the lab.

Read it here: [Single-cell reconstruction of developmental trajectories during zebrafish embryogenesis](https://www.ncbi.nlm.nih.gov/pubmed/29700225)

Also, check out a perspective article about our research: [&quot;A new view of embryo development and regeneration&quot;](https://science.sciencemag.org/content/360/6392/967.long)

This work was eventually featured as part of <i>Science</i> magazine's [Breakthrough of the Year for 2018](../12/science-boty.html).